src/ray-controller.js

```diff
@@ -125,19 +128,19 @@ export default class RayController extends EventEmitter {
   }
 
   update() {
-    let mode = this.getInteractionMode();
-    if (mode == InteractionModes.VR_3DOF || mode == InteractionModes.VR_6DOF) {
-      // If we're dealing with a gamepad, check every animation frame for a
-      // pressed action.
-      let isGamepadPressed = this.getGamepadButtonPressed_();
-      if (isGamepadPressed && !this.wasGamepadPressed) {
-        this.emit('raydown');
-      }
-      if (!isGamepadPressed && this.wasGamepadPressed) {
-        this.emit('rayup');
-      }
-      this.wasGamepadPressed = isGamepadPressed;
+    if (!this.gamepad)
```

Comment:
Um, no... we still want to raycast if there's no gamepad. Please test on all supported devices.